scripts/rename-template.mjs

```diff
@@ -89,8 +89,7 @@ const performRename = async () => {
   const mappingsFile = './scripts/config/mappings.json'
   const originalMappings = await fs.readFile(mappingsFile, "utf8");
   let mappings = await JSON.parse(originalMappings);
-  mappings.SERVERLESS_DOMAIN.name = `serverless_functions_domain_${packageSuffixUndercore}`
-  mappings.VIDEO_SERVERLESS_DOMAIN.name = `serverless_functions_domain_${packageSuffixUndercore}`
```

Comment:
did you mean to delete this? it seems unrelated to this PR and i dont see any associated changes

Reply:
Yup! This was removed from mappings.json in a previous PR and I failed to remove it here.